### DISCUSSION TITLE
ScriptReference/Collider.OnTriggerEnter

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#702] [Notes]
# ScriptReference/Collider.OnTriggerEnter

https://docs.unity3d.com/ScriptReference/Collider.OnTriggerEnter.html

**Comment by laurentopia** [comment]
If the "other" trigger has a RigidBody either in its GameObject or that of its parents, then that trigger will be detected by OnTriggerEnter.
